ts/src/test/test.ts

```diff
@@ -355,8 +355,9 @@ export default class testMainClass extends baseMainTestClass {
             }
             // we don't throw exception for public-tests, see comments under 'testSafe' method
             let failedMsg = '';
-            if (errors.length) {
-                failedMsg = ' | Failed methods: ' + errors.join (', ');
+            const len = errors.length;
```

Comment:
you can't call it len because it is the name of a function in python

Reply:
ahh, I didn't even notice it. pushed the fix.